bin/registerServerTopology.sh

```diff
@@ -433,16 +433,11 @@ function request_to_remote() {
 function set_remote_parameters_manually() {
     # ask information to connect to Remote API
     echo "A few more information are required to convert your platform into Remote : "
-    echo "${CURRENT_NODE_ADDRESS} : Please enter your username:"
-    read -r API_CURRENT_NODE_USERNAME
-    echo -n "Please enter the password of $CURRENT_NODE_ADDRESS: "
-    API_CURRENT_NODE_PASSWORD=$(get_api_password)
-    echo -e "\n${CURRENT_NODE_ADDRESS} : Protocol [http]:"
-    read -r INPUT_PROTOCOL
-    echo "${CURRENT_NODE_ADDRESS} : Port [80]:"
-    read -r INPUT_PORT
-    echo "${CURRENT_NODE_ADDRESS} : centreon root folder [centreon]:"
-    read -r INPUT_CENTREON_FOLDER
+    read -p "${CURRENT_NODE_ADDRESS} : Please enter your username: " API_CURRENT_NODE_USERNAME
+    read -sp "Please enter the password of $CURRENT_NODE_ADDRESS: " API_CURRENT_NODE_PASSWORD; echo ""
```

Comment:
`echo` is needed ?

Reply:
it handle the carriage return, as \n is not interpreted in read